langchain/src/cache/fs.ts

```diff
@@ -0,0 +1,70 @@
+import { BaseCache, Generation } from "../schema/index.js";
+import { getCacheKey } from "./base.js";
+import path from "path";
+import fs from "node:fs/promises";
+
+const DEFAULT_CACHE_DIR = path.join(process.cwd(), ".langchain-cache");
```

Comment:
Makes more sense to use something like this if not specified?

https://github.com/langchain-ai/langchainjs/blob/80fea35b8d4dceb64ad4394271977e8d94777455/langchain/src/storage/tests/file_system.test.ts#L11

Reply:
I think this one is up to what you prefer as a default behaviour. In my case, I want the cache to survive the end of the process execution. But both situation are fully possible. So if you confirm you prefer a temp dir, let's do it.

Reply:
I'd prefer tempdir as a default then 👍